### PR TITLE
cmake: openthread: Fix library copy target for openthread.

### PR DESCRIPTION
**Pull request by canisLupus1313**
Fix for changed path of mbedtls config.

### CODE REVIEW

**Review by maciejbaczmanski — APPROVED** [review]

File: openthread/CMakeLists.txt [hunk 1]
```diff
@@ -23,7 +23,7 @@ if(CONFIG_OPENTHREAD_SOURCES)
   set(OPENTHREAD_UTILS_DST_DIR "${NRFXLIB_DIR}/openthread/include/utils")
   set(OPENTHREAD_SYSTEM_H "${ZEPHYR_OPENTHREAD_MODULE_DIR}/examples/platforms/openthread-system.h")
   set(OPENTHREAD_UART_H "${ZEPHYR_OPENTHREAD_MODULE_DIR}/examples/platforms/utils/uart.h")
-  set(NRF_SECURITY_MBEDTLS_CONFIG_FILE ${CMAKE_CURRENT_BINARY_DIR}/../nrf_security/src/include/generated/${CONFIG_MBEDTLS_CFG_FILE})
+  set(NRF_SECURITY_MBEDTLS_CONFIG_FILE ${CMAKE_CURRENT_BINARY_DIR}/../../../nrf/subsys/nrf_security/src/include/generated/${CONFIG_MBEDTLS_CFG_FILE})
```
Comment (nordicjm) [inline openthread/CMakeLists.txt:26]: Why can't this refer to the nrf module variable instead?

**Review by nordicjm — CHANGES_REQUESTED** [review]

File: openthread/CMakeLists.txt (hunk 1 above)
Comment (canisLupus1313) [reply openthread/CMakeLists.txt:26]: which one ?

Comment (nordicjm) [reply openthread/CMakeLists.txt:26]: `${ZEPHYR_NRF_MODULE_DIR}`

Comment (canisLupus1313) [reply openthread/CMakeLists.txt:26]: This unwinds on my env to `$HOME/Repos/ncs/nrf/` and I want to copy build generated mbedtls config file.

Comment (nordicjm) [reply openthread/CMakeLists.txt:26]: Oh it's a generated file, will approve, my bad

**Review by nordicjm — APPROVED** [review]

**Pull request merged**